src/utils.ts

```diff
@@ -62,15 +62,16 @@ export const moveElementsByN = <T>(n: number, arr: T[]) =>
 
 export const formatSelectedDate = (
   selectedDate: Date | Date[] | null | undefined,
-  dateFormat: string
+  dateFormat: string,
+  formatOptions: Object | null,
```

Comment:
Same here, please reuse the type created and turn it into an optional param.

Reply:
Done